examples/website/src/shared/blocks/peerPageBreak.tsx

```diff
@@ -6,6 +6,7 @@ import {cssRule, useStyle} from '@karma.run/react'
 import {Color} from '../style/colors'
 import {pxToRem, whenMobile} from '../style/helpers'
 import {Link} from '../route/routeContext'
+import {RichTextBlockValue} from '@wepublish/editor/src/client/blocks/types'
```

Comment:
No cross package imports

Reply:
Checkout how the richText Block handles this

Reply:
corrected